src/screens/ConfigurationScreen.tsx

```diff
@@ -1,21 +1,53 @@
-import * as React from 'react';
-import { StyleSheet } from 'react-native';
+import { default as React, useState } from 'react';
+import { StyleSheet, View } from 'react-native';
 import { ScrollView } from 'react-native-gesture-handler';
-import OptionButton from '../components/OptionButton';
 import Colors from '../constants/Colors';
+import VentSwitch from '../components/VentSwitch';
+import ConfigSelect from '../components/ConfigSelect';
+import configs from '../constants/Configurables';
+import ToggleUpdate from '../components/UpdateButton';
+
+const INITIAL_STATE = {
```

Comment:
general naming convention is `camelCase`, isn't it? How come you used `INITIAL_STATE` instead of `initialState`?

Reply:
@themakshter please check [this](https://google.github.io/styleguide/javascriptguide.xml?showone=Constants#Constants)

Reply:
This has been discussed offline and it makes sense to a degree. It is used in Redux and there are good reasons behind it: https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/.

The only thing I would say would be that in terms of consistency with the rest of the application code, you should move this to the `constants` folder. You can overwrite the stuff in the `InitialVentilatorConfiguration.ts` with this and import it here.